script/s3.js

```diff
@@ -141,8 +142,34 @@ const recursiveRead = (dir, done) => {
   });
 };
 
+const persistDeployTime = deployTime => {
+  const metricsInfo = awsCredentials.metrics || {};
+  if (metricsInfo.baseUrl) {
+    const axiosApi = axios.create({
+      baseURL: metricsInfo.baseUrl,
+      timeout: 10000
+    });
+    body = {
+      env,
+      tech: metricsInfo.tech,
+      repo_name: metricsInfo.repoName,
+      metrics: [
+        {
+          name: 'deploy-time',
+          version: '1.0',
+          value: `${deployTime}`
+        }
+      ]
+    }
+    axiosApi.post('/metrics', body).then(() => console.log('Deploy time saved succesfully'), error => console.log(`Deploy time couldn't be saved error: ${error}`));
+  } else {
+    console.log("Deploy time couldn't be saved due to the missing api base url")
```

Comment:
```suggestion
    console.log('Deploy time couldn't be saved due to the missing api base url')
```

Reply:
I have a `couldn't` in the middle. That's why i use double quotes